test/test_manifest.json

```diff
@@ -4702,5 +4702,11 @@
       "md5": "f84ce8b7414f6a18e75a6ce69c902501",
       "rounds": 1,
       "type": "eq"
+    },
+    {  "id": "issue12337",
+       "file": "pdfs/issue12337.pdf",
+       "md5": "9165772d5b860bcbcc2478f32e311eb0",
+       "rounds": 2,
+       "type": "fbf"
```

Comment:
Do you actually need to test all pages, since I'd imagine that using `lastPage: 1` should work!?

Given that `page.cleanup` is always run between pages, note [the `completeRender` function](https://github.com/mozilla/pdf.js/blob/741ce4f7fc451473c45b7a42ccc4138c94234e14/test/driver.js#L625-L645), that *should* be enough to trigger the bug since it's indirectly what's happening when the page is evicted from the `PDFPageViewBuffer` in the viewer.

Reply:
You're right, that works!